anaconda/meta.yaml

```diff
@@ -18,6 +18,5 @@ requirements:
 about:
     home: https://www.snowflake.com/
     license: Apache 2.0
-    license_file: /tmp/anaconda_workspace/src/LICENSE.txt
-    summary: Snowflake Telemetry Python package 
-
+    license_file: {{ environ.get('SNOWFLAKE_TELEMETRY_DIR') }}/LICENSE.txt
```

Comment:
Not sure if we can do this in yaml file, as we normally do in Python script

Reply:
This meta.yaml file is processed by the `conda build ./anaconda --output-folder ./anaconda/dist` and these substitutions are processed correctly. This is tested in this Jenkins job: https://ci-dev-142.int.snowflakecomputing.com/job/SnowflakeTelemetryAnacondaPackageBuilder/